addons/io_scene_gltf2/__init__.py

```diff
@@ -854,6 +854,18 @@ class ImportGLTF2(Operator, ImportHelper):
         default=True
     )
 
+    merge_vertices: BoolProperty(
+        name='Merge Vertices',
+        description=(
+            'On export to glTF, vertices get split up to create pieces '
+            'with continuous normals/UVs/etc.\n'
+            'When this option is on, attempt to "undo" this and merge '
+            'those verts back together.\n'
+            'Currently cannot merge verts with different normals'
```

Comment:
It's not just Blender export that's the issue.  How about...

> The glTF format requires discontinuous normals, UVs, and other vertex attributes to be stored as separate vertices, as required for rendering on typical graphics hardware. This option attempts to combine co-located vertices where possible.

Reply:
Changed.